src/neuroglancer/single_mesh/frontend.ts

```diff
@@ -27,7 +27,7 @@ import {getObjectId} from 'neuroglancer/util/object_id';
 import {Uint64} from 'neuroglancer/util/uint64';
 import {withSharedVisibility} from 'neuroglancer/visibility_priority/frontend';
 import {Buffer} from 'neuroglancer/webgl/buffer';
-import glsl_COLORMAPS from 'neuroglancer/webgl/colormaps.glsl';
+import {glsl_COLORMAPS} from 'neuroglancer/webgl/colormaps.ts';
```

Comment:
Don't use a .ts extension in the module imports.

Reply:
Fixed